xray/commands/audit/sca/npm/npm.go

```diff
@@ -31,6 +39,18 @@ func BuildDependencyTree(params utils.AuditParams) (dependencyTrees []*xrayUtils
 
 	treeDepsParam := createTreeDepsParam(params)
 
+	var restoreNpmrcFunc func() error
+	restoreNpmrcFunc, err = configureResolutionServerIfNeeded(params)
+	if err != nil {
+		log.Warn(fmt.Sprintf("Configuring an Artifactory server failed. Dependencies will be resolved from NPM default registry if needed\nFailure cause: %s", err.Error()))
+		err = nil
+	}
```

Comment:
I'm not sure that the code should decide whether to skip resolution from Artifactory based on an error.

Reply:
deprecated